packages/components/src/components/SquigglePlayground.tsx

```diff
@@ -374,52 +384,80 @@ export const SquigglePlayground: FC<PlaygroundProps> = ({
     </div>
   );
 
+  let withEditor = (
+    <div className="flex mt-1" style={{ height }}>
+      <div className="w-1/2">
+        <Tab.Panels>
+          <Tab.Panel>
+            <div className="border border-slate-200">
+              <CodeEditor
+                value={squiggleString}
+                onChange={setSquiggleString}
+                oneLine={false}
+                showGutter={true}
+                height={height - 1}
+              />
+            </div>
+          </Tab.Panel>
+          <Tab.Panel>{samplingSettings}</Tab.Panel>
+          <Tab.Panel>{viewSettings}</Tab.Panel>
+          <Tab.Panel>{inputVariableSettings}</Tab.Panel>
+        </Tab.Panels>
+      </div>
+
+      <div className="w-1/2 p-2 pl-4">
+        <SquiggleChart
+          squiggleString={squiggleString}
+          environment={env}
+          chartSettings={chartSettings}
+          height={vars.chartHeight}
+          showTypes={vars.showTypes}
+          showControls={vars.showControls}
+          showSummary={vars.showSummary}
+          bindings={defaultBindings}
+          jsImports={imports}
+        />
+      </div>
+    </div>
+  );
+
+  let withoutEditor = (
+    <div className="mt-3">
+      <Tab.Panels>
+        <Tab.Panel>
+          <SquiggleChart
+            squiggleString={squiggleString}
+            environment={env}
+            chartSettings={chartSettings}
+            height={vars.chartHeight}
+            showTypes={vars.showTypes}
+            showControls={vars.showControls}
+            bindings={defaultBindings}
+            jsImports={imports}
+            showSummary={vars.showSummary}
+          />
```

Comment:
Maybe avoid copy-paste with intermediate `firstPanelContents` and `panels` vars?